core/lib/multistore.ts

```diff
@@ -148,19 +148,25 @@ export function localizedDispatcherRoute (routeObj: LocalizedRoute | string, sto
     return appendStoreCodePrefix ? '/' + storeCode + routeObj : routeObj
   }
 
-  if (routeObj && routeObj.fullPath) { // case of using dispatcher
+  if (routeObj && routeObj.path) { // case of using dispatcher
```

Comment:
@lukeromanowicz please make sure this is safe - meaning there is no other call to `localizedRoute()` in the theme, using the `path` in case no `urlDispatcher` is needeed; I now remember: we've used fullPath to distict the use cases and be sure they're exclusively for urlDispatcher